src/components/kronisk/KroniskSide.tsx

```diff
@@ -247,7 +242,6 @@ const KroniskSide = () => {
                     })}
 
                     <Textarea
-                      label='annet'
```

Comment:
Denne skal visuelt fjernes, men at fjerne den breaker med a11y reglene. Ideas?

Reply:
Be Tommy om å skjerpe seg. Kan ikke bryte a11y reglene!